src/handlers/cell.ts

```diff
@@ -207,6 +202,19 @@ export class CellManager implements IDisposable {
     }
   };
 
+  private addBreakpointsToEditor(cell: CodeCell) {
+    this.clearGutter(cell);
+    const editor = cell.editor as CodeMirrorEditor;
+    const breakpoints = this._debuggerModel.breakpointsModel.breakpoints;
+    breakpoints.forEach(breakpoint => {
+      editor.editor.setGutterMarker(
+        breakpoint.line - 1,
```

Comment:
Not super happy of these `+/-1` lying around. Would be better to have a proper way to map that information.